bluemira/utilities/plot_tools.py

```diff
@@ -116,7 +119,7 @@ def make_gif(folder, figname, formatt="png", clean=True):
             if filename.endswith(formatt):
                 fp = os.path.join(folder, filename)
                 ims.append(fp)
-    ims = sorted(ims)
+    ims = sorted(ims, key=lambda x: int(re.sub("\\D", "", x)))
```

Comment:
@je-cook please take a look; I suck at regex. The idea is to get the trailing number in a list of filenames and sort based on increasing order. This is effectively to avoid adding natsort as a dependency.

Reply:
It looks fine but maybe find existing instead of not existing might be better? This makes sure we get the last set of digits eg 'f1231ig_123.png' -> 123:
```suggestion
    find_digit = re.compile("(\d+)")
    ims = sorted(ims, key=lambda x: int(find_digit.findall(x)[-1]))
```

Reply:
the only hole I can see in both is if there are no numbers or numbers in weird places but probably catching those is overkill

Reply:
Thanks!